drivers/can/can_native_linux.c

```diff
@@ -445,13 +449,21 @@ static int can_native_linux_init(const struct device *dev)
 {
 	const struct can_native_linux_config *cfg = dev->config;
 	struct can_native_linux_data *data = dev->data;
+	const char *if_name;
 
 	k_mutex_init(&data->filter_mutex);
 	k_sem_init(&data->tx_idle, 1, 1);
 
-	data->dev_fd = linux_socketcan_iface_open(cfg->if_name);
+	if (if_name_cmd_opt != NULL) {
+		if_name = if_name_cmd_opt;
+	} else {
+		if_name = cfg->if_name;
+	}
+	LOG_DBG("Opening %s", if_name);
+	data->dev_fd = linux_socketcan_iface_open(if_name);
+
```

Comment:
```suggestion
	data->dev_fd = linux_socketcan_iface_open(if_name);
```

Reply:
I don't see what changes you want me to do. I can't even click "commit suggestion" because it says that there are no changes

Reply:
I'd like you to remove the empty line between assignment and test to logically group the two together. Guess GH couldn't represent it without further lines in the context, sorry.

Reply:
Done